src/screens/SingleSpeakerScreen.tsx

```diff
@@ -0,0 +1,205 @@
+                            {myloop}
+                        </Text>
+                    </View>
+                    <View style={styles.sectiontwo_subsectiontwo}>
+                        <Text style={styles.sectiontwo_subsectiontwo_header}>Bio</Text>
+                        <ScrollView>
+                        <Text style={styles.sectiontwo_subsectiontwo_paragraph}>
+                            {textElements}
+                        </Text>
+                        </ScrollView>
+                        
+                    </View>
+                </View>
+                <View style={styles.sectionthree}>
+                    <Text style={styles.sectionthree_text}>Twitter Handle</Text>
+                    <TouchableOpacity onPress={() => {alert("Twitter handle pressed!")}}>
+                    <View style={styles.sectionthree_subsectionone}>
+                        <TwitterIcon width={30} height={25} color={colors.DROIDCONKE_BLUE}/>
+                        <Text style={{...styles.sectionthree_text, color: colors.DROIDCONKE_BLUE}}>{item[0].speakerhandle}</Text>
+                    </View>
+                    </TouchableOpacity>
+                </View>
+            </View>
+        </SafeAreaView>
+    )
+};
+
+export default SingleSpeakerScreen;
+
+const styles = StyleSheet.create({
+    container: {
+        width: "100%",
+        height: "100%", 
+        display: "flex",
+    },
+    sectionone: {
+        flex: 2,
+        position: "relative"
+    },
+    bgimage: {
+        flex: 1
+    },
+    sectiontwo: {
+        backgroundColor: colors.DROIDCONKE_WHITE,
+        flex: 7
+    },
+    sectionthree: {
+        marginTop: 1,
+        backgroundColor: colors.DROIDCONKE_WHITE,
+        flex: 1,
+        flexDirection: "row",
+        justifyContent: "space-between",
+        alignItems: "center",
+        paddingHorizontal: 10,
+    },
+    sectionthree_text: {
+        fontSize: 16,
+        fontFamily: fonts.MONTSERRAT_REGULAR,
+    },
+    arrowleft: {
+        backgroundColor: "red",
+        width: 50, 
+        height: 50
+    },
+    profilepic: {
+       // backgroundColor: "blue",
+        width: 100, 
+        height: 100,
+        position: "absolute",
+        left: "37%",
+        top: "75%",
+        zIndex: 1,
+        borderRadius: 50, 
+        borderWidth: 3,
+        borderColor: colors.DROIDCONKE_LIGHT_GREEN
+    },
+    sectiontwo_subsectionone: {
+        alignItems: "center",
+        paddingTop: 65
+    },
+    sectiontwo_subsectionone_itemone: {
+        color: colors.DROIDCONKE_BRICK_RED,
+        fontSize: 15,
+        fontFamily: fonts.MONTSERRAT_REGULAR
+    },
+    sectiontwo_subsectionone_itemtwo: {
+        color: colors.DROIDCONKE_BLUE,
+        fontSize: 20,
+        marginBottom: 10,
+        fontFamily: fonts.MONTSERRAT_BOLD,
+        padding: 0,
+        lineHeight: 20
+    },
+    sectiontwo_subsectionone_itemthree: {
+      //  fontSize: 15,
```

Comment:
You can delete this line in case you will not use it.

Reply:
Sorry I forgot